hello-nextflow/hello-workflow.nf

```diff
@@ -1,4 +1,21 @@
 #!/usr/bin/env nextflow
+/*
+ * Pipeline parameters
+ */
+params.greeting = 'greetings.csv'
+
+workflow {
+
+    // create a channel for inputs from a CSV file
+    greeting_ch = Channel
+        .fromPath(params.greeting)
+        .splitCsv()
+        .map { line -> line[0] }
+
+    // emit a greeting
+    sayHello(greeting_ch)
+}
```

Comment:
the `format` command now will not try to re-order your declarations unless you enable it, should make the formatting less intrusive overall